app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -363,7 +369,9 @@ private void takeScreenshotOfContextMenu(UiDevice device) throws UiObjectNotFoun
         multiTabBtn.click();
         eraseHistoryBtn.waitForExists(waitingTime);
         Screengrab.screenshot("Multi_Tab_Menu");
-        eraseHistoryBtn.click();
```

Comment:
the definition of eraseHistoryBtn is incorrect, at least on tablet mode. Uiautomator will pick whatever element that matches first, and there are more than one element that is android.widget.TextView class with index of 2. (namely, the url bar) If we prefer to erase it this way, let's assign an identifier to the erase history button.  As a workaround, I deleted the tabs from the notification bar.